JavaBasicProgram/src/StringMethod/stringExampleMethod.java

```diff
@@ -0,0 +1,97 @@
+package StringMethod;
+
+public class stringExampleMethod {
+    public static void main(String[] args) {
+        // TODO Auto-generated method stub
+        //charAt
+        String s = "Excellarate";
```

Comment:
Please follow the points shared in teams

Reply:
done